### PR TITLE
#281 - Add a count for customers created per day

### PR DESCRIPTION
**Pull request by jonathanyeong**
### Description
Adds methods in Reporting to get the number of customers created per day.

Also:
- Add a customer test_changeset. This allows us to modify the `inserted_at` column in tests.
- Add tests for count_customers_by_date methods. Separated from the other test blocks because
  we don't need to create a customer during the setup.

### Issue
https://github.com/papercups-io/papercups/issues/281

## Checklist

- [x] Everything passes when running `mix test`
- [x] Ran `mix format`
- [x] No frontend compilation warnings

### CODE REVIEW

File: lib/chat_api/customers/customer.ex [hunk 1]
```diff
@@ -102,4 +102,10 @@ defmodule ChatApi.Customers.Customer do
       :time_zone
     ])
   end
+
+  def test_changeset(message, attrs) do
+    message
```
Comment (reichert621) [inline lib/chat_api/customers/customer.ex:107]: nit -- let's change `message` to `customer`

**Review by reichert621 — COMMENTED** [review]
looks great @jonathanyeong! :)  

just one small nitpick and then should be good to merge 🎉

File: lib/chat_api/customers/customer.ex (hunk 1 above)
Comment (jonathanyeong) [reply lib/chat_api/customers/customer.ex]: Argh copy and paste error. Good catch thanks!

**Review by reichert621 — APPROVED** [review]
thanks @jonathanyeong! 😄

**Pull request merged**